cmd/security-agent/main_windows.go

```diff
@@ -174,6 +178,10 @@ func (s *service) Run(svcctx context.Context) error {
 		}),
 
 		statusimpl.Module(),
+
+		fetchonlyimpl.Module(),
+		configsyncimpl.OptionalModule(),
+		fx.Invoke(func(_ optional.Option[configsync.Component]) {}),
```

Comment:
What is the purpose for this ?

Reply:
The component is not used as a dependency by any other component, so `fx` won't start it unless it's used by some `invoke`
We do the same thing for other components, eg: https://github.com/DataDog/datadog-agent/blob/741fc7974cefff0cffb994e37911fbfc91ec5b35/cmd/agent/subcommands/run/command_windows.go#L200-L203

Reply:
Would you mind adding a comment like in the link you shared ?